source/faq.txt

```diff
@@ -19,8 +19,8 @@ Frequently Asked Questions
 
 This page contains frequently asked questions and their answers.
 
-Why Do I Get ``ModuleNotFoundError: No module named 'polars'`` When Using {+driver-short+}?
----------------------------------------------------------------------------------------
+Why do I get ``ModuleNotFoundError: No module named 'polars'`` when using {+driver-short+}?
```

Comment:
I: [Titles need headline-style caps](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#capitalization)

```suggestion
Why Do I Get ``ModuleNotFoundError: No module named 'polars'`` When Using {+driver-short+}?
```